app/views/granules/download.html.erb

```diff
@@ -1,19 +1,90 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <title><%= site_page_title('Downloads') %></title>
-    <link rel="shortcut icon" href="/favicon.ico?v=1.1">
-  </head>
-  <body>
-    <ul>
-      <% @urls.each do |url| %>
-      <li><a href="<%= url %>"><%= url %></a></li>
-      <% end %>
-    </ul>
-    <% if  @errors.present? %>
-      <%= @errors %>
-     <% end %>
-  </body>
+<head>
```

Comment:
Steals @bilts ```cwic_granule``` code. However, to find all download links, we can't query CMR directly but have to go through our rails app to parse links for opendap granules.